src/directory/directory.js

```diff
@@ -1788,6 +1788,11 @@ const directory = {
             title: 'Build options',
             route: '/cli/function/build-options',
             filters: []
+          },
+          {
+            title: 'Configuring Lambda function settings',
```

Comment:
This seems like a long title name, does this render properly? Is "Configure Options" too vague? 

Additionally, we should aim to keep the filename and titles similar

Reply:
After I posted this I noticed `'Calling DynamoDB using AWS Cognito triggers'` below